content/community/moderation.md

```diff
@@ -47,6 +47,8 @@ Example responsibilities include:
 - Accepting and rejecting changes to the documentation
 - Serving as intermediaries between users and admins
 
+List of moderator features can be seen on the [moderator features](/references/moderator-features/) reference page.
```

Comment:
```suggestion
```

Reply:
> Moderators are a small team of active users selected by admins and are responsible for supervising users' activities. They have earned the trust of the admins from years of contributions and are given access to [moderation tools] and extra privileges.

Reply:
Done. Updated the link in the related blog post too.